packages/corvid-cli/src/commands/clone.js

```diff
@@ -44,6 +49,15 @@ async function cloneHandler(args) {
     });
 }
 
+async function clearFiles(rootPath, dirContentsBefore) {
```

Comment:
clearFiles is not a good enough name imo

Reply:
also, can you unite "clearFiles" with "dirContentsBefore" into a nicer "container" that is fully responsible for this logic ?

Reply:
done